### PR TITLE
Matsuri 2021 Base Update

### PR DESCRIPTION
**Pull request by 20wildmanj**
- Hid away unnecessary pages (Events, Sponsors, Infographic, various Matsuri pages)
- Updated Member page and email contact
- Cleaned up Matsuri Section and added baseline content (waiting on revisions to clean up content)
- Added Culture Placeholder pages

### CODE REVIEW

File: src/routes/events/events.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 title,startDate,endDate,location,contentPath
-Matsuri 2019,2019-04-23,2019-04-23,Carnegie Mellon University,/matsuri/
+Matsuri 2021,2021-04-09,2019-04-11,Carnegie Mellon University & Virtual,/matsuri/
```
Comment (aavondet) [inline src/routes/events/events.csv:2]: 2019->2021

File: src/routes/matsuri/cause/index.md [hunk 2]
```diff
@@ -1,9 +1,3 @@
...
-
-The organization was established as the nation's first private social welfare campaign after WWII, starting with helping those affected by the war. Through the years, the community chest has helped many people in Japan, collaborating closely with local people to effectively support as many people as efficiently as possible. To this date, the organization has received a total of 943.3 billion Japanese Yen, which helped increase the scope of impact it has had on the people living in Japan. 
-
-We decided to donate our money raised from Matsuri to this organization as many many people, including members of CMU JSA, were greatly impacted by the 2011 Tohoku earthquake.
-
-Learn more at: https://www.akaihane.or.jp.e.gt.hp.transer.com/
\ No newline at end of file
+This year we chose to hold Matsuri with two causes in mind, helping the local Japanese restaurant community in Pittsburgh and raising awareness about the Asian identity in the United States. In the past we have held the event for causes in Japan, but we made the choice this year to focus on issues that affect our local community instead.
```
Comment (aavondet) [inline src/routes/matsuri/cause/index.md:3]: Note: Eventually get Simon to write something better, maybe he already has?

File: src/routes/matsuri/games/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- 4/9-4/11
+- Submissions will be open from 9:00 am - 9:30 pm EDT
+- There will be an announcement posted on social media (Facebook, Instagram) at 9:00 am about the day's 2-3 challenges
+- There will be posts announcing point totals after each day of the challenge
+
+
+## Synchronous Games
+Main Times (EDT)
+- 4/9 Friday 19:00 - 20:00 
+- 4/10 Saturday 15:00 - 16:00
```
Comment (aavondet) [inline src/routes/matsuri/games/index.md:20]: I think we chose to just do the escape room + trivia in the middle of the performances?

**Review by aavondet — CHANGES_REQUESTED** [review]
For the most part looks good, make sure not to commit your .DS_Store files though, I saw several scattered across the directories. Make sure you delete those, check my comments, and I think you can merge after that 🚀

File: src/routes/matsuri/cause/index.md (hunk 2 above)
Comment (20wildmanj) [reply src/routes/matsuri/cause/index.md:3]: Yep I sent him a message to check the main page and the cause page to revise and clean it up, along with the other exec members in charge of the other aspects like food etc.

File: src/routes/matsuri/games/index.md (hunk 3 above)
Comment (20wildmanj) [reply src/routes/matsuri/games/index.md:20]: I asked Jonathan and he didn't have any issues with the times, you should probably talk to him about it

Comment (aavondet) [reply src/routes/matsuri/games/index.md:20]: huh I thought we chose to merge them with the performances...interesting

**Pull request merged**